src/app/services/recent-activity.ts

```diff
@@ -11,41 +11,12 @@ export class RecentActivityService {
 
   constructor(private api: ApiService) { }
 
-  // getByTantalisId(tantalisId: number): Observable<RecentActivity[]> {
-  //   return this.api.getRecentActivitysByTantalisId(tantalisId)
-  //     .map(res => {
-  //       if (res && res.length > 0) {
-  //         const features: Array<RecentActivity> = [];
-  //         res.forEach(f => {
-  //           features.push(new RecentActivity(f));
-  //         });
-  //         return features;
-  //       }
-  //       return [];
-  //     })
-  //     .catch(error => this.api.handleError(error));
-  // }
-
-  // getByProjectId(projectId: string): Observable<RecentActivity[]> {
-  //   return this.api.getRecentActivitysByProjectId(projectId)
-  //     .map(res => {
-  //       if (res && res.length > 0) {
-  //         const features: Array<RecentActivity> = [];
-  //         res.forEach(f => {
-  //           features.push(new RecentActivity(f));
-  //         });
-  //         return features;
-  //       }
-  //       return [];
-  //     })
-  //     .catch(error => this.api.handleError(error));
-  // }
-
-  // deleteByApplicationId(applicationId: string): Observable<Object> {
-  //   return this.api.deleteRecentActivitysByApplicationId(applicationId)
-  //     .catch(error => this.api.handleError(error));
-  // }
-
+  /**
+   * Get a recetn activity by its ID.
+   *
+   * @param {string} activityId The recent activity ID to get by.
+   * @returns {Observable}
+   */
   getById(activityId: string) {
```

Comment:
It looks like some return types aren't declared in this file. Maybe that's beyond the scope of this ticket though.